app-rails/README.md

```diff
@@ -96,6 +96,15 @@ To run natively:
 1. `make start-native`
 1. Then visit http://localhost:3100
 
+#### Local Authentication
+
+You can circumvent typical login and log in using any email and password if you add
+```
+AUTH_ADAPTER=mock
```

Comment:
Since the default for local is already mock, maybe flip the instruction here to explain how to set to cognito (and explain that you also need to set the other cognito env vars)